recipe/meta.yaml

```diff
@@ -53,6 +54,9 @@ test:
     - pytest
   commands:
     - pytest --pyargs mpas_analysis
+    - mpas_analysis --help
+    - download_analysis_data --help
+    - download_nautral_earth_110m --help
```

Comment:
typo in "natural" here too

Reply:
Copy-paste at its worst!  The typo is also in `setup.py` but everyone uses `conda-forge` so I don't think that's as big a deal.  I'll fix it but I won't do a new release for it...

Reply:
Yeah, This one was hard to see -- I just read right over it multiple times.

Seems fair to wait on a bugfix release for it to actually break on someone if you expect everyone to be using conda-forge. :-D